### ISSUE TITLE
[Feature]:  Importing commands to index.js needs to be changed

### ISSUE THREAD

**Issue by Tan-Shadow** [commclassroom/classroom-monitor-bot#74]
### Is your feature request related to a problem? Please describe.

So if a command is added we'll have to use `const command = require( 'commands/command' )`
This is will be bad once the code starts growing

### Describe the solution you'd like

there is a better way to add commands, like this we'll have to add every single command with `const ... = require(...)`
instead what we can do is
```js
const commandFiles = fs
    .readdirSync('./commands/')
    .filter(file => file.endsWith('.js'));
for (const file of commandFiles) {
    const command = require(`./commands/${file}`);

    client.commands.set(command.name, command);
}
``` 
then use 
```js
client.on( 'message',  msg => {
    if (
        !message.content.startsWith(prefix) ||
        message.author.bot ||
        message.channel.type == 'dm' // this solves one of the issue that i have raised
    )
        return; 

  const args = message.content.slice(prefix.length)
  const command = args.shift().toLowerCase();  
  
  else if ( command == "/command/" ) client.get(/command/).execute(msg, args, Discord)
  // same for other commands
})
```

**Comment by kaiwalyakoparkar** [comment]
Hey, @Tan-Shadow Nice thought. Would you like to go ahead with this issue?

**Comment by Tan-Shadow** [comment]
yes but i would love to, but i will need a bit of help

**Comment by Tan-Shadow** [comment]
Nevermind...i solved it made a pr

**Comment by Panquesito7** [comment]
Is this already fixed or still being worked on it?

**Comment by Tan-Shadow** [comment]
Fixed by #76
